.dockerignore

```diff
@@ -0,0 +1,4 @@
+bin/*
+
+!bin/server/**
+!bin/scratch/cert-manager.license
```

Comment:
note: this means "ignore everything in `bin` except these two paths", which are the only ones currently consumed by the Containerfile